core/src/trezor/ui/layouts/tt_v2/__init__.py

```diff
@@ -666,11 +758,13 @@ async def confirm_metadata(
         layout = trezorui2.show_warning(
             title="Unusually high fee",
             description=param or "",
+            button="CONTINUE",
```

Comment:
any reason `"CONTINUE"` is not the default value on Rust side?

Reply:
no reason, fixed in 7c9cc2343fd569e349c7412ff2a4761decac9a4d